packages/augur-state/src/IsJsonRpcRequest.ts

```diff
@@ -0,0 +1,6 @@
+export function isJsonRpcRequest(message: any): boolean {
+  if (message.jsonrpc !== "2.0") return false;
```

Comment:
You may also want to test that `message` isn't `undefined` since then `message.jsonrpc` will give an error.

Reply:
Added (even though github isn't showing that right now).